language-plutus-napkin/src/Language/PlutusCore/Identifier.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+module Language.PlutusCore.Identifier ( -- * Types
+                                        IdentifierState
+                                      , Unique (..)
```

Comment:
Wouldn't it be better to export `Unique` abstractly? (We don't want anybody poking around in its representation outside of this module, do we?)